u19_pipeline/recording.py

```diff
@@ -0,0 +1,181 @@
+    'NEW_SESSION':       {'Value': 0,
+                         'Label': 'New session',
+                         'Task_Field': None},
+    'RAW_FILE_REQUEST':  {'Value': 1,
+                          'Label': 'Raw file transfer requested',
+                          'Task_Field': 'task_copy_id_pre_path'},
+    'RAW_FILE_CLUSTER':  {'Value': 2,
+                         'Label': 'Raw file transferred to cluster',
+                         'Task_Field': None},
+    'JOB_QUEUE':         {'Value': 3,
+                         'Label': 'Processing job in queue',
+                         'Task_Field': 'slurm_id_sorting'},
+    'JOB_FINISHED':      {'Value': 4,
+                         'Label': 'Processing job finished',
+                         'Task_Field': None},
+    'PROC_FILE_REQUEST': {'Value': 5,
+                         'Label': 'Processed file transfer requested',
+                         'Task_Field': 'task_copy_id_pos_path'},
+    'PROC_FILE_HOME':    {'Value': 6,
+                         'Label': 'Processed file transferred to PNI',
+                         'Task_Field': None},
+    'CANONICAL_PIPELINE': {'Value': 7,
+                         'Label': 'Processed with Canonical pipeline',
+                         'Task_Field': None},
+}
+
+def get_content_list_from_status_dict():
+    contents = list()
+    for i in status_pipeline_dict.keys():    
+        contents.append([status_pipeline_dict[i]['Value'], status_pipeline_dict[i]['Label']])
+    return contents
+
+
+@schema
+class StatusProcessDefinition(dj.Lookup):
+     definition = """
+     status_pipeline:                    TINYINT(1)      # status in the automate process pipeline
+     ---
+     status_definition:                  VARCHAR(256)    # Status definition 
+     """
+     contents = get_content_list_from_status_dict()
+
+
+@schema
+class RecordingProcess(dj.Manual):
+     definition = """
+     recording_process_id:              INT(11) AUTO_INCREMENT    # Unique number assigned to each processing job for a recording 
+     -----
+     -> Recording
+     -> StatusProcessDefinition                                   # current status in the pipeline
+     -> PreprocessingParamSet                                     # reference to params to preprocess recording
+     -> ProcessingParamSet                                        # reference to params to process recording
+     recording_process_path=null:       VARCHAR(200)              # relative path for processed recording
+     task_copy_id_pre=null:             UUID                      # id for globus transfer task raw file cup->tiger  
+     task_copy_id_pos=null:             UUID                      # id for globus transfer task sorted file tiger->cup
+
+     slurm_id=null:                     VARCHAR(16)               # id for slurm process in tiger
+     """    
+
+
+@schema
+class RecordingProcessStatus(dj.Manual):
+     definition = """
+     ->RecordingProcess
```

Comment:
```suggestion
     -> RecordingProcess
```

Reply:
Could there be a one-to-many relationship between the `RecordingProcess` and `RecordingProcessStatus` tables?